<?xml version="1.0"?>
<launch>

 <!-- joy node -->
  <node respawn="true" pkg="joy" type="joy_node" name="joy" >
    <param name="dev" type="string" value="/dev/input/js0" />
    <param name="deadzone" value="0.12" />
  </node>

  <!-- Parameters -->
  <param name="cmd_topic_vel" value="/cmd_vel" type="string"/>
  <param name="cmd_service_io" value="modbus_io/write_digital_output" type="string"/>
  <param name="cmd_topic_ptz" value="rb1_base_robot_control/command_ptz" type="string"/>
  
  <rosparam command="load" file="$(find rb1_base_joystick)/launch/ps3.yaml" />
  <node name="rb1_base_joystick" pkg="rb1_base_joystick" type="rb1_base_joystick" output="screen"/>
  
</launch>

